S2VX.Game.Tests/HeadlessTests/ScoreProcessorTests/ScoreStatisticsTests.cs

```diff
@@ -0,0 +1,140 @@
+﻿using NUnit.Framework;
+using osu.Framework.Allocation;
+using osu.Framework.Testing;
+using S2VX.Game.Play.UserInterface;
+using S2VX.Game.Story;
+using S2VX.Game.Story.Note;
+
+namespace S2VX.Game.Tests.HeadlessTests.ScoreProcessorTests {
+    [HeadlessTest]
+    public class ScoreStatisticsTests : S2VXTestScene {
+        [Cached]
+        private S2VXStory Story { get; } = new();
+        private Notes Notes { get; set; }
+        private ScoreProcessor Processor { get; } = new();
+
+        [BackgroundDependencyLoader]
+        private void Load() => Add(Processor);
+
+        [SetUpSteps]
+        public void SetUpSteps() {
+            // Tried to put this line into Load() but ran into an issue where if
+            // I ran a test by itself through the Test Explorer, the Load()
+            // wasn't triggered correctly
```

Comment:
I'm still getting an error when running a single test
![image](https://user-images.githubusercontent.com/12549910/121446968-8a004080-c949-11eb-9f2f-858f6066bc05.png)

Reply:
That looks like it's for other ScoreProcessor tests. I'll go and fix the other tests.

Reply:
Refactored other files